UXO_Kismet_SF_3.tex

```diff
@@ -0,0 +1,363 @@
+\end{itemize}
+% end contents subsection -- }}}
+
+% begin personnel required section -- {{{
+\subsection{Personnel Required}
+The test operations team consists of nine personnel:
+\begin{checklist}
+    \item The \opsfull{} directs operations procedures and communicates with the other test personnel.
+    \item The \primaryfull{} operates all manual valves for the fill system.
+    \item The \secondaryfull{} is the backup for \primary{}, and communicates with OPS. If \primary{} becomes incapacitated, \secondary{} is responsible for removing them from danger.
+    \item The \daqfull{} monitors and operates the test data acquisition system.
+    \item the \controlfull{} operates the test control system, including actuation of remote valves and engine ignition.
+    \item The \heatfull{} operates the valves for the tank heating system.
+    \item \perifull{}, \periifull{}, and \periiifull{} ensure that no unauthorized personnel enter the testing area during test operations.
+\end{checklist}
+\setcounter{checklistnum}{0}
+% end personnel required section -- }}}
+
+% begin sign off -- {{{
+\subsection{Sign-Off}
+\textit{To be completed by all test personnel after reading and familiarization with procedures}
+\begin{checklist}
+    \item \opsfull{}      \tabto{25em}\rule{10em}{0.4pt}\hspace{5em}\rule{10em}{0.4pt}
+    \item \primaryfull{}  \tabto{25em}\rule{10em}{0.4pt}\hspace{5em}\rule{10em}{0.4pt}
+    \item \secondaryfull{}\tabto{25em}\rule{10em}{0.4pt}\hspace{5em}\rule{10em}{0.4pt}
+    \item \daqfull{}      \tabto{25em}\rule{10em}{0.4pt}\hspace{5em}\rule{10em}{0.4pt}
+    \item \controlfull{}  \tabto{25em}\rule{10em}{0.4pt}\hspace{5em}\rule{10em}{0.4pt}
+    \item \heatfull{}     \tabto{25em}\rule{10em}{0.4pt}\hspace{5em}\rule{10em}{0.4pt}
+    \item \perifull{}     \tabto{25em}\rule{10em}{0.4pt}\hspace{5em}\rule{10em}{0.4pt}
+    \item \periifull{}    \tabto{25em}\rule{10em}{0.4pt}\hspace{5em}\rule{10em}{0.4pt}
+    \item \periiifull{}   \tabto{25em}\rule{10em}{0.4pt}\hspace{5em}\rule{10em}{0.4pt}
+\end{checklist}
+\setcounter{checklistnum}{0}
+% end sign off -- }}}
+
+\newpage
+% begin prior to start section -- {{{
+\subsection{Prior to Start}
+\begin{checklist}
+    \item Ensure that the following procedures are complete:
+    \begin{checklist}
+        \item Combustion Chamber Assembly procedure
+        \item Oxidizer Tank Assembly procedure
+        \item Plumbing Setup procedure
+        \item Oxidizer Tank Stand Setup procedure
+        \item Tank Heating Setup procedure
+        \item Test Stand Setup procedure
+        \item Data Acquisition Setup procedure
+        \item Test Control System Setup procedure
+    \end{checklist}
+    \item Ensure that all personnel as defined above are available and have completed the sign-off.
+    \item Ensure that the following personnel have walkie-talkies and communication is functional:
+    \begin{checklist}
+        \item \ops{}
+        \item \secondary{}
+        \item \daq{}
```

Comment:
Usually DAQ has a walkie because 1) they're doing a lot of the talking (pressure readouts etc.), and 2) if we need to troubleshoot, check sensor readings, etc., they'll need to communicate with secondary